cerebras_tools_proxy.py

```
@@ -0,0 +1,562 @@
+#!/usr/bin/env python3
+"""
+Tool-Enabled Cerebras Proxy Server
+Combines Cerebras Cloud API integration with full tool execution capabilities
+Designed for Claude Code CLI integration with Cerebras 480B model.
+"""
+
+import os
+import sys
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)